render/root.go

```diff
@@ -80,6 +81,13 @@ func (r Root) Paint(solidBackground bool, opts ...RootPaintOption) []image.Image
 		r.maxFrameCount = DefaultMaxFrameCount
 	}
 
+	if r.FrameWidth != 0 {
```

Comment:
Maybe something like:
```
frameWidth := DefaultFramWidth
if r.FrameWidth != 0 {
    frameWidth = r.FrameWidth
}
```

Sorry if this feels like a nit, I find it really confusing to have a default that mutates 😅

Reply:
No worries, golang is not my primarily language so any feedback is great!